From linux-usb-devel-bounces@lists.sourceforge.net  Sat Sep  8 07:49:13 2007
From: "samson yeung" <fragmede@onepatchdown.net>
Date: Fri, 31 Aug 2007 16:40:40 -0400
Subject: usb/misc/sisusbvga: add product ID of TARGUS/MCT device
To: linux-usb-devel@lists.sourceforge.net
Message-ID: <c038f5910708311340va29c404jb3447a64061afbc9@mail.gmail.com>
Content-Disposition: inline


Device is Targus ACP50US which includes a Magic Control Technologies
usb vga device using the SiS315(E) or compatible.

Signed-off-by: Samson Yeung <fragmede@onepatchdown.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/misc/sisusbvga/sisusb.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/misc/sisusbvga/sisusb.c
+++ b/drivers/usb/misc/sisusbvga/sisusb.c
@@ -3404,6 +3404,7 @@ static void sisusb_disconnect(struct usb
 }
 
 static struct usb_device_id sisusb_table [] = {
+	{ USB_DEVICE(0x0711, 0x0550) },
 	{ USB_DEVICE(0x0711, 0x0900) },
 	{ USB_DEVICE(0x0711, 0x0901) },
 	{ USB_DEVICE(0x0711, 0x0902) },
